source/layouts/layout.erb

```diff
@@ -73,7 +73,7 @@ under the License.
     <div class="page-wrapper">
       <div class="dark-box">
         <div class="ribbon right green">
-          <a href="https://github.com/sproutbox/cheddargetter-api-docs">Fork me on GitHub</a>
```

Comment:
This should stay as cheddargetter for now but the link is actually this since the Github org was changed to `chdr`:

https://github.com/chdr/cheddargetter-api-docs